vision/recognition.proto

```diff
@@ -0,0 +1,27 @@
+
+syntax = "proto3";
+
+package admobilize_vision;
+
+// Describes a recognition feature descriptor
+message FeatureDescriptor {
+
```

Comment:
If you decide to leave a space after every message definition you should be consistent. You can check the other messages and adjust.